src/telemetry/exporters.ts

```diff
@@ -1,160 +1,72 @@
-import { Client, QueryConfig, QueryArrayResult, PoolConfig } from "pg";
-import { groupBy } from "lodash";
-import { LogMasks, DBOSDataType, MethodRegistrationBase } from "./../decorators";
-import { DBOSPostgresExporterError } from "./../error";
-import { TelemetrySignal } from "./signals";
+import { TelemetrySignal } from "./collector";
+import { isLogSignal, isTraceSignal } from "./";
 import { OTLPTraceExporter } from "@opentelemetry/exporter-trace-otlp-http";
-import { ReadableSpan } from "@opentelemetry/sdk-trace-base";
+import { OTLPLogExporter } from "@opentelemetry/exporter-logs-otlp-http";
+import type { ReadableSpan } from "@opentelemetry/sdk-trace-base";
+import type { ReadableLogRecord } from '@opentelemetry/sdk-logs';
 import { ExportResult, ExportResultCode } from "@opentelemetry/core";
-import { spanToString } from "./traces";
 
-export interface ITelemetryExporter<T, U> {
-  export(signal: TelemetrySignal[]): Promise<T>;
-  process?(signal: TelemetrySignal[]): U;
-  init?(registeredOperations?: ReadonlyArray<MethodRegistrationBase>): Promise<void>;
-  destroy?(): Promise<void>;
+export interface OTLPExporterConfig {
+  logsEndpoint?: string;
+  tracesEndpoint?: string;
 }
 
-export const JAEGER_EXPORTER = "JaegerExporter";
-export class JaegerExporter implements ITelemetryExporter<void, undefined> {
-  private readonly exporter: OTLPTraceExporter;
-  constructor(endpoint: string = "http://localhost:4318/v1/traces") {
-    this.exporter = new OTLPTraceExporter({
-      url: endpoint,
-    });
+export interface ITelemetryExporter {
+  export(signal: TelemetrySignal[]): Promise<void>;
+}
+
+export class TelemetryExporter implements ITelemetryExporter {
```

Comment:
Do we have to change the configuration file at all to support this new one-exporter, two-endpoint structure?

Reply:
Nope, the current way of loading the configuration supports this change transparently.

Reply:
Sorry if I wasn't clear: the update to the configuration types is done [here](https://github.com/dbos-inc/dbos-ts/blob/0fe544b62e6215accf415272fcabfe4068f0ee15/src/telemetry/index.ts#L21)